nxontology_ml/efo.py

```diff
@@ -1,106 +1,57 @@
-import functools
 from collections import Counter
-from pathlib import Path
-from typing import Any
 
-from nxontology_ml.features import NodeInfoFeatures, NxontologyFeatures
-from nxontology_ml.utils import ROOT_DIR, get_output_directory
+import numpy as np
+from nxontology.node import NodeInfo
 
+from nxontology_ml.sklearn_transformer import DataFrameFnTransformer
 
-class NodeInfoFeaturesEfo(NodeInfoFeatures[str]):
-    @staticmethod
-    def get_curie_prefix(curie: str) -> str:
-        prefix, _identifier = curie.split(":", 1)
-        return prefix.lower()
 
-    @property
-    def prefix(self) -> str:
-        curie = self.identifier
-        assert isinstance(curie, str)
-        return self.get_curie_prefix(curie)
+def _get_curie_prefix(identifier: str | None) -> str:
```

Comment:
should we keep the argument here as `curie` for more consistency where `curie` is like `prefix:identifier`

Reply:
Since the change was very small, I forced pushed the changes. [Patch visible here](https://github.com/related-sciences/nxontology-ml/compare/02c99ccb13c063dc6a9e094d22221c1b2435c973..44c5b3de73a115bc117748d2571223486577fecb).